package.json

```diff
@@ -16,8 +16,8 @@
     "compilemessages": "formatjs compile",
     "format": "prettier --write 'src/openforms/{js,scss}/**/*.{js,scss,mdx}'",
     "checkformat": "prettier --check 'src/openforms/{js,scss}/**/*.{js,scss,mdx}'",
-    "storybook": "start-storybook -p 6006",
-    "build-storybook": "build-storybook"
+    "storybook": "storybook dev -p 6006 -no-version-updates",
```

Comment:
```suggestion
    "storybook": "storybook dev -p 6006 --no-version-updates",
```
?

Reply:
I've found the relevant config file option via the TS definitions, so this is obsolete :)